core/src/components/chip/chip.scss

```diff
@@ -120,6 +120,21 @@
   @include margin(-4px, -8px, -4px, 8px);
 }
 
+// Chip Shapes
+// ---------------------------------------------
+
+:host(.chip-soft) {
+  --border-radius: 8px;
```

Comment:
This was previously set to `16px` but I checked the [Material Design documentation](https://m3.material.io/components/chips/specs#590903f7-2bf5-46ab-9810-d052173f41f1) and they use `8px` in their latest guidelines so I decided to update it here.

Reply:
Verified that it's 8px.